integration/kubernetes/init.sh

```diff
@@ -115,6 +116,14 @@ kubectl apply -f "$network_plugin_config"
 # we need to ensure a few specific pods ready and running
 wait_pods_ready
 
+# Add 'cniVersion' to the flannel configuration file.
+# Fix needed just for flannel to let cni release ip addresses,
+# we should remove after https://github.com/coreos/flannel/pull/1135
+# gets merged and we update flannel_version above.
```

Comment:
If we've not already, maybe we should have a kata issue to track, so we don't forget (we'll get reminded when the stalebot starts whining about it...)